### PR TITLE
updated readme for installing java (jdk), setting up environmental variables, and install java 17

### PR DESCRIPTION
**Pull request by AustinWongParker**

### CODE REVIEW

**Review by MitchellTFord — COMMENTED** [review]
Windows-specific instructions 😠

**Comment by AustinWongParker** [comment]
pls approve lord mitchell @MitchellTFord

**Review by AustinWongParker — COMMENTED** [review]
windows > all operating systems

**Comment by MitchellTFord** [comment]
I am not going to approve a PR that has a "coming soon" in it LOL

**Comment by AustinWongParker** [comment]
:( mitchell tyrant

**Comment by MitchellTFord** [comment]
We aren't in the 90's anymore, Austin. We can't put "Coming Soon" and "Under Construction" on our MySpace pages anymore

**Comment by AustinWongParker** [comment]
I'm a 90s baby, so I get a pass on it. Don't worry

File: README.md [hunk 1]
```diff
@@ -1,6 +1,19 @@
...
+# Set Environmental Variables - JDK
+Navigate to environmental variables -> system variables (bottom pane) -> left-click path -> edit -> new -> paste in your path (example: C:\Program Files\Java\YOUR_JDK_HERE\bin)
+
+# Set JAVA_HOME
+Navigate back to System variables -> new -> variable name is JAVA_HOME -> variable value is C:\Program Files\Java\YOUR_JDK_HERE
+  ```
+### Install Java (JDK) & Set-up Environmental Variables (windows)
+```shell
+coming soon
+```
```
Comment (MitchellTFord) [inline README.md:15]: Remove this because this isn't the 90's anymore
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -1,6 +1,19 @@
 # discord-bot
 
 ## Getting Started
+### Install Java (JDK) & Set-up Environmental Variables (windows)
```
Comment (MitchellTFord) [inline README.md:4]: Capitalize Windows because it has feelings too
```suggestion
### Install Java (JDK) & Set-up Environmental Variables (Windows)
```

**Review by MitchellTFord — COMMENTED** [review]
okay do these things and I will accept, okay? see? not a tyrant

**Review by MitchellTFord — APPROVED** [review]
Approved :+1
@CodeMasterBrandon will be contacting you shortly to thank you for your service

**Pull request merged**